### DISCUSSION TITLE
Improving Docker Image

### DISCUSSION THREAD

**Discussion by rahulsurwade08** [kubescape/kubescape#700] [Ideas]
I have some suggestions for the docker image to improve its build size and security.
We can use [distroless](https://github.com/GoogleContainerTools/distroless) base images in multistage build which provide both reduction in build size and contain less vulnerabilities.

**Comment by rahulsurwade08** [comment]
@dwertent @slashben Can I get some feedback on this discussion?

**Comment by slashben** [comment]
Hey @RahulSurwade08 , we were thinking about this in the past. However we got the impression from the users, that they prefer to have tools in the container (bash and etc.)

**Reply by rahulsurwade08** [reply]
Hey @slashben , Thanks for the feedback.
